maintainers/flake-module.nix

```diff
@@ -12,6 +12,8 @@
       hooks = {
         clang-format = {
           enable = true;
+          # https://github.com/cachix/git-hooks.nix/pull/532
+          package = pkgs.llvmPackages_latest.clang-tools;
```

Comment:
What's the impact of this on the devshell closure size? (It's already pretty big, depending on gcc, clang and a lot of other stuff.)

Reply:
Same as the current clang-tools version. But we need both macOS and Linux to use the same version, open for other suggestions, but increase the closure size on some platform for sure. I sticked to _latest because it will never go out-of-date compared to pinning a version.

Reply:
Not having this will block our nixpkgs upgrade.